README.md

```diff
@@ -0,0 +1,394 @@
+
```

Comment:
Title: My start page
Date: 2024-04-05 20:23:40
Modified: 2024-04-05 20:23:40
Author: Carlos Ibarra 
Tags: OMN default, Index

* [Quick notes](QuickNotes.html)
* [Tags](Tags.html)
* [Incoming pages](incoming/Incoming.html)
* [Incoming bookmarks](bookmarks/Incoming.html)
* [Changelog](default/Changelog.html)
* [Welcome!](default/Welcome.html)
* [Help](default/Help.html)
* [Extended Markdown syntax](default/MarkdownExt.html)
* [Build information](default/Build.html)

**This page generated automatically. Edit this as you wish.**